filters/shedder/admission.go

```diff
@@ -331,6 +331,9 @@ func (ac *admissionControl) Close() {
 func (ac *admissionControl) tickWindows(d time.Duration) {
 	t := time.NewTicker(d)
 	i := 0
+	defer func() {
+		t.Stop()
+	}()
```

Comment:
Why not `defer t.Stop()`?

Reply:
yeah true left over because I tried to close(t.C) but that did not work because of it's only receiver side